docs/self-managed/operational-guides/update-guide/850-to-860.md

```diff
@@ -30,6 +30,10 @@ The default value of Elasticsearch deployment pods has changed from 2 to 3, and
 If one of your endpoints returns multiple Set-Cookie headers and you need to capture all of them, set `groupSetCookieHeaders` to `true` in the element template XML. This aggregates the headers into a list. This feature is available since version 8.6.7. The grouping is enabled by default since version 8.6.11.
 :::
 
+:::caution Breaking change
+If one of your endpoints require pre-encoded URL elements this will break with 8.6.0. Since 8.6.12. the `skipEncoding` in the XML value can be set to `"true"`. This disables the automatic decoding and re-encoding process, ensuring the URL is sent to the server exactly as provided.
```

Comment:
We could rephrase it and mention the `skipEncoding` property of the element template itself.

Reply:
@ztefanie Wording looks good now. Customers use element-templates (and need to upgrade them) to make use of new features and they rarely change the code in the XML itself.

Reply:
@ztefanie Got you!